### DISCUSSION TITLE
BigBlueButton-Integration

### DISCUSSION THREAD

**Discussion by linux-lukas** [pretalx/pretalx#1182] [General]
I could imagine that it would minimise a lot of work if BBB rooms could be created from Pretalx for talks, if necessary at online or hybrid events, as Nextcloud already has this functionality.

**Comment by rixx** [comment]
This seems like an excellent candidate for a pretalx plugin!

**Reply by rixx** [reply]
(Also, of course: Using venueless with BBB integration and pretalx integration would probably make for a nicer, more discoverable experience.)
